packages/editor/src/store/selectors.js

```diff
@@ -427,6 +427,29 @@ export function isEditedPostBeingScheduled( state ) {
 	return date.isAfter( now );
 }
 
+/**
+ * Returns whether the current post should be considered to have a "floating"
+ * date (i.e. that it would publish "Immediately" rather than at a set time).
+ *
+ * Unlike in the PHP backend, the REST API returns a full date string for posts
+ * where the 0000-00-00T00:00:00 placeholder is present in the database. To
+ * infer that a post is set to publish "Immediately" we check whether the date
+ * and modified date are the same.
+ *
+ * @param  {Object}  state Editor state.
+ *
+ * @return {boolean} Whether the edited post has a floating date value.
+ */
+export function isEditedPostDateFloating( state ) {
```

Comment:
I expect this new selector to generate some changes in the docs if you run `npm run docs:build`. Can you commit these changes?

Reply:
> I expect this new selector to generate some changes in the docs if you run `npm run docs:build`. Can you commit these changes?

This was never done and there are local changes to commit on master after running the command.

Please review #10234

Reply:
:grimacing: my bad, @aduth , hadn't had time to circle back to this and had missed that piece of feedback. I'll know for next time.

Reply:
No worries @kadamwhite